### ISSUE TITLE
"Close all tabs" also closes pinned tabs

### ISSUE THREAD

**Issue by voidl** [JannesMeyer/TabAttack#18]
Should the "Close all tabs" button also close pinned tabs that weren't imported into TabAttack?

![image](https://cloud.githubusercontent.com/assets/7597143/7756669/da696abe-ffca-11e4-98a4-ba72e5a26729.png)

**Comment by JannesMeyer** [comment]
That's quite true! I hadn't thought of that. And it probably shouldn't close tabs that were ignored based on the domain name neither.

A fix for this will definitely make it into the next release. But I'm currently working in a separate branch on the next version which will have Firefox support which is only 50% done so far.

**Comment by voidl** [comment]
Great! thanks for the awesome extension.

**Comment by sknebel** [comment]
@JannesMeyer are patches to the current version of the extension likely to be portable to the new one, or should I wait until the version with firefox support is out? (I have a few ideas for stuff to add, but nothing that needs to be added ASAP)
